.github/workflows/Regression.yml

```diff
@@ -130,10 +130,12 @@ jobs:
         python scripts/regression_test_runner.py --old=duckdb/build/release/benchmark/benchmark_runner --new=build/release/benchmark/benchmark_runner --benchmarks=.github/regression/csv.csv --verbose --threads=2
 
     - name: Regression Test new micro benchmark
-      if: always()
+      if: github.event_name == 'pull_request'
```

Comment:
I wouldn't do this on pull request. It means this only triggers when the pull request is opened, but not when a user pushes updates to their branch